docs/releases/1.26-NOTES.md

```diff
@@ -27,6 +27,10 @@ with "control-plane-". The names of groups for existing clusters are unchanged.
 
 * New clusters can more easily be configured to use Cilium in ENI mode by setting `--networking=cilium-eni`.
 
+## GCP
+
+* The default instance type is now `e2-medium` for control plane and nodes, and `e2-micro` for bastions.
```

Comment:
```suggestion
* The default instance type is now `e2-medium` for control-plane and worker nodes, and `e2-micro` for bastions.
```

Reply:
I can't see that much usage of `control-plane` spelling on https://kubernetes.io.

Reply:
Hyphenation of compound modifiers is one of the lesser-well-known English grammar rules.